docs/usage/configuration-options.md

```diff
@@ -1514,7 +1514,8 @@ For example you have multiple `package.json` and want to use `dependencyDashboar
 ```
 
 Important to know: Renovate will evaluate all `packageRules` and not stop once it gets a first match.
-You should order your `packageRules` in order of importance so that later rules can override settings from earlier rules if needed.
+You should order your `packageRules` in order of ascending importance so that more important rules come later
```

Comment:
I'm not a native English speaker, but my sense of language tells me I'd say "in ascending order of importance" instead. And maybe use "increasing" instead of "ascending"?

Reply:
Thanks for the input - you are right; "in ascending order of importance" does sound better. I've updated this PR to say that. I kept "ascending" instead of "increasing", but happy to change it if you prefer.